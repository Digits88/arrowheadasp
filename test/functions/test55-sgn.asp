<!-- Test Sgn function.
-->

Sgn(0) = <%= Sgn(0) %><br>
Sgn(1) = <%= Sgn(1) %><br>
Sgn(-1) = <%= Sgn(-1) %><br>
Sgn(123) = <%= Sgn(123) %><br>
Sgn(-212) = <%= Sgn(-212) %><br>
