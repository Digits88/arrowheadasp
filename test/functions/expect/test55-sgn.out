<!-- Test Sgn function.
-->

Sgn(0) = 0<br>
Sgn(1) = 1<br>
Sgn(-1) = -1<br>
Sgn(123) = 1<br>
Sgn(-212) = -1<br>
